source/linux/background-managers/hyprpaper.js

```diff
@@ -0,0 +1,39 @@
+import childProcess from 'node:child_process';
+import {commandExists, execFile} from '../util.js';
+
+export async function isAvailable() {
+	return commandExists('hyprpaper');
+}
+
+async function initialize() {
+	return new Promise(resolve => {
+		const hyprpaper = childProcess.spawn('script', ['-q', '-c', 'hyprpaper', '/dev/null']);
+		hyprpaper.stdout.on('data', data => {
+			if (data.toString().includes('[ERR]') || data.toString().includes('error')) {
+				throw new Error(data.toString());
+			} else {
+				resolve();
+			}
+		});
+	});
+}
+
+export async function get() {
+	await initialize();
```

Comment:
Are you sure it actually needs to initialize first?

Reply:
Yes. hyprpaper can use its config to set wallpaper at startup.